src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -19,7 +19,7 @@
 	public void loadSomeData() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)